plugins/telescope/ui-select.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.telescope.extensions.ui-select;
+in {
+  options.plugins.telescope.extensions.ui-select = {
+    enable = mkEnableOption "ui-select extension for telescope";
+
+    package = helpers.mkPackageOption "telescope extension ui-select" pkgs.vimPlugins.telescope-ui-select-nvim;
+
+    settings = mkOption {
```

Comment:
Is `settings` a good name for that ?
I chose it because it fitst RFC42.

Reply:
I'd say yes, let's try to follow more closely RFC-42